### PR TITLE
Bug 1394362 - Allow font scaling in Onboarding slides.

### PR DESCRIPTION
**Pull request by farhanpatel**

### CODE REVIEW

File: Client/Frontend/Intro/IntroViewController.swift [hunk 1]
```diff
@@ -8,6 +8,8 @@ import SnapKit
 struct IntroViewControllerUX {
     static let Width = 375
     static let Height = 667
+    static let SyncButonTopPadding = 5
```
Comment (garvankeeley) [inline Client/Frontend/Intro/IntroViewController.swift]: s/buton/button

File: Client/Frontend/Intro/IntroViewController.swift [hunk 2]
```diff
@@ -8,6 +8,8 @@ import SnapKit
...
+    static let minimumFontScale = 0.5
```
Comment (garvankeeley) [inline Client/Frontend/Intro/IntroViewController.swift]: do we capitalize these (so far i just followed prevailing style in the existing file i was in)?

File: Client/Frontend/Intro/IntroViewController.swift [hunk 3]
```diff
@@ -189,6 +194,13 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
             make.bottom.equalTo(syncCardView)
         }
 
+        // We need a refrence to the sync pages textlabel so we can adjust the constraints
```
Comment (garvankeeley) [inline Client/Frontend/Intro/IntroViewController.swift]: s/refrence/reference/

File: Client/Frontend/Intro/IntroViewController.swift [hunk 4]
```diff
@@ -189,6 +194,13 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
...
+        if let index = introViews.index(of: syncCardView), index < textLabels.count {
+            let syncTextLabel = textLabels[index]
+            syncTextLabel.snp.makeConstraints { make in
+                make.bottom.equalTo(signInButton.snp.top).offset(-IntroViewControllerUX.SyncButonTopPadding)
```
Comment (garvankeeley) [inline Client/Frontend/Intro/IntroViewController.swift]: I am surprised we didn't have this constraint before (I didn't read the rest of the layout code, I would have thought the bottom of that label would already be glued to the top of the view below it).

File: Client/Frontend/Intro/IntroViewController.swift (hunk 3 above)
Comment (farhanpatel) [reply Client/Frontend/Intro/IntroViewController.swift]: I need a spell checker for my comments 😢

File: Client/Frontend/Intro/IntroViewController.swift (hunk 4 above)
Comment (farhanpatel) [reply Client/Frontend/Intro/IntroViewController.swift]: The whole vc is pretty wonky. But it works!

**Pull request merged**